dbux-code/src/commands/applicationsViewCommands.js

```diff
@@ -18,6 +20,14 @@ export function initApplicationsViewCommands(context) {
     stopRuntimeServer
   );
 
+  registerCommand(context,
+    'dbuxApplicationsView.clearApplication',
+    () => {
+      allApplications.clear();
```

Comment:
UX basics: whenever deleting something, or making other types of dangerous edits, check with the user in a modal first